packages/server/src/create-blitz-app.ts

```diff
@@ -0,0 +1,60 @@
+import next from "next"
+import {build} from "./build"
+import {alreadyBuilt} from "./build-hash"
+import {normalize, ServerConfig} from "./config"
+import {configureStages} from "./stages"
+interface CreateBlitzAppConfig {
+  dev: boolean
+}
+
+export async function createBlitzApp({dev}: CreateBlitzAppConfig) {
+  const serverConfig: ServerConfig = {
+    env: dev ? "dev" : "prod",
+    rootFolder: process.cwd(),
+    // port
+    // hostname
+    // inspect
+    // clean: flags["no-incremental-build"],
+  }
+  const {
+    rootFolder,
+    transformFiles,
+    devFolder,
+    ignore,
+    include,
+    isTypescript,
+    writeManifestFile,
+    watch,
+    clean,
+    buildFolder,
+  } = await normalize({...serverConfig, env: "dev"})
+
+  if (dev) {
+    // dev
+    const stages = configureStages({writeManifestFile, isTypescript})
+
+    await transformFiles(rootFolder, stages, devFolder, {
+      ignore,
+      include,
+      watch,
+      clean,
+    })
```

Comment:
In `server/dev.ts` we have this:

https://github.com/blitz-js/blitz/blob/75b63ed087265db268b2c0f9025c2377ae270da4/packages/server/src/dev.ts#L21-L30

**🙋‍♂️  Two questions regarding this**

- What is the purpose of `readyForNextDev`?
- What is `manifest` used for and is it something I need to consider here?